python/lsst/ts/standardscripts/scheduler/base_standby.py

```diff
@@ -0,0 +1,47 @@
+# This file is part of ts_standardscripts.
+#
+# Developed for the LSST Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+__all__ = ["BaseStandBy"]
+
+from lsst.ts import salobj
+
+from .base_set_desired_state import BaseSetDesiredState
+
+
+class BaseStandBy(BaseSetDesiredState):
```

Comment:
I strongly suggest eliminating all subclasses of BaseSetDesiredState and just implementing the trivial subclasses in the script files.

If you make the desired state a constructor parameter and compute the descr in the base class, that looks like this:
```
from lsst.ts.idl.enums.Scheduler import SalIndex
from lsst.ts.salobj import State
from lsst.ts.standardscripts.scheduler import SetDesiredState

class Standby(SetDesiredState):
    def __init__(self, index):
        super().__init__(index=index, desired_state=State.STANDBY, scheduler_index=SalIndex.AUX_TEL)

asyncio.run(Standby.amain())
```

If I update BaseScript.amain to support kwargs, this simplifies to:
```
from lsst.ts.idl.enums.Scheduler import SalIndex
from lsst.ts.salobj import State
from lsst.ts.standardscripts.scheduler import SetDesiredState

asyncio.run(SetDesiredState.amain(desired_state=State.STANDBY, scheduler_index=SalIndex.AUX_TEL))
```